Core/include/Acts/Geometry/GeometryObject.hpp

```diff
@@ -71,19 +71,19 @@ class GeometryObject {
 
   /// Set the value
   ///
-  /// @param geoID the geometry identifier to be assigned
-  void assignGeoID(const GeometryID& geoID);
+  /// @param geometryId the geometry identifier to be assigned
+  void assigngeometryId(const GeometryID& geometryId);
```

Comment:
I think this should then be named `assignGeometryId(...)`

Reply:
See below, otherwise It would not compile, I fear.

Reply:
Fixing this.